emma-language/src/main/scala/org/emmalanguage/io/csv/CSV.scala

```diff
@@ -18,32 +18,53 @@ package io.csv
 
 import io.Format
 
-import java.nio.charset.{Charset, StandardCharsets}
+import java.nio.charset.Charset
+import java.nio.charset.StandardCharsets
 
 import CSV._
 
-/** CSV format. */
-case class CSV
-(
-  //@formatter:off
-  header    : Boolean      = DEFAULT_HEADER,     // Indicate the presence of a CSV header (to be ignored)
-  delimiter : Char         = DEFAULT_DELIMITER,  // Column delimiter character
-  charset   : Charset      = DEFAULT_CHARSET,    // Character set
-  quote     : Option[Char] = DEFAULT_QUOTE,      // Delimiters inside quotes are ignored
-  escape    : Option[Char] = DEFAULT_ESCAPE,     // Escaped quote characters are ignored
-  comment   : Option[Char] = DEFAULT_COMMENT,    // Disable comments by setting this to None
-  nullValue : String       = DEFAULT_NULLVALUE   // Fields matching this string will be set as nulls
-  //@formatter:on
-) extends Format
+/**
+ * CSV format configuration.
+ * @param header     Indicate the presence of a CSV header (to be ignored when reading).
+ * @param delimiter  Column delimiter character.
+ * @param delimitSeq Sequence delimiter character (used by CSV converters for collections).
+ * @param charset    Character set.
+ * @param quote      Delimiters inside quotes are ignored.
+ * @param escape     Escaped characters are always treated as data.
+ * @param comment    Lines starting with a comment character are ignored.
+ * @param joinHeader Specifies how to flatten the headers of nested case classes.
```

Comment:
Will remove it. I considered generating a header from case class field names via `LabelledGeneric` (and then you would need to flatten nested headers something like `address.city`), but we don't have any use for this.